src/test/java/org/mskcc/cbio/oncokb/service/CompanyServiceIT.java

```diff
@@ -346,4 +348,121 @@ public void assertThatCompanyUsersFound(){
         assertThat(companyUsers.get(0).getCompany().getId()).isEqualTo(company.getId());
     }
 
+    @Test
+    @Transactional
+    public void assertThatTrialUserExpirationWillBeExtendedAfterLinkedWithTrialCompany() {
+        userService.createUser(userDTO, Optional.empty(), Optional.empty());
+
+        Optional<User> userOptional = userService.initiateTrialAccountActivation(userDTO.getLogin());
+        assertThat(userOptional.isPresent()).isTrue();
+
+        User user = userOptional.get();
+        Optional<UserDetailsDTO> userDetailsDTO = userDetailsService.findOneByUser(user);
+        userService.finishTrialAccountActivation(userDetailsDTO.get().getAdditionalInfo().getTrialAccount().getActivation().getKey());
+
+        Instant userTokenExpirationDate = Instant.now().plusSeconds(1000);
+        tokenService.findByUser(user).forEach(token -> {
+            token.setExpiration(userTokenExpirationDate);
+            tokenService.save(token);
+        });
+
+        // Create ta trial company
```

Comment:
Minor typo also in the next 3 tests
```suggestion
        // Create a trial company
```